testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -241,6 +270,90 @@ impl BlockMinerThread {
         }
     }
 
+    fn wait_for_signed_block(
+        &self,
+        stackerdbs: &StackerDBs,
+        aggregate_public_key: &Point,
+        signer_signature_hash: &Sha512Trunc256Sum,
+    ) -> Option<NakamotoBlock> {
+        let stackerdb_contracts = stackerdbs
+            .get_stackerdb_contract_ids()
+            .expect("FATAL: could not get the stacker DB contract ids");
+        // TODO: get this directly instead of this jankiness when .signers is a boot contract
```

Comment:
Not sure what this comment means?

Reply:
I shouldn't have to verify that the miner contains the signers_contract_id. I wrote this before it even existed. This enabled the tests to pass for both my own where I manually deployed signers vs ones that did not.

Reply:
Ah gotcha